editoast/src/client/runserver.rs

```diff
@@ -43,6 +43,10 @@ pub struct RunserverArgs {
     // only recieve 401 responses.
     #[clap(long, env = "EDITOAST_DISABLE_AUTHORIZATION", default_value_t = true)]
     disable_authorization: bool,
+    /// If this option is set, logging for the STDCM will be enabled.
+    /// When enabled, relevant logs will be captured to aid in debugging and monitoring.
+    #[clap(long, env = "ENABLE_STDCM_LOG", default_value_t = true)]
+    enable_stdcm_log: bool,
```

Comment:
nit: would

```suggestion
    enable_stdcm_logging: bool,
```

or

```suggestion
    log_stdcm: bool,
```

be more correct?

Reply:
Done.